server/data/prisonApiClient.ts

```diff
@@ -7,20 +7,17 @@ import {
   OffenderRestrictions,
   CaseLoad,
 } from './prisonApiTypes'
-import config from '../config'
+import config, { ApiConfig } from '../config'
 
-export const prisonApiClientBuilder = (token: string): PrisonApiClient => {
-  const restClient = new RestClient('prisonApi', config.apis.prison, token)
-  const prisonClient = new PrisonApiClient(restClient)
+export default class PrisonApiClient {
+  private restClient: RestClient
 
-  return prisonClient
-}
-
-class PrisonApiClient {
-  constructor(private readonly restclient: RestClient) {}
+  constructor(token: string) {
+    this.restClient = new RestClient('prisonApiClient', config.apis.prison as ApiConfig, token)
+  }
 
-  getBookings(offenderNo: string, prisonId: string): Promise<PagePrisonerBookingSummary> {
-    return this.restclient.get({
+  async getBookings(offenderNo: string, prisonId: string): Promise<PagePrisonerBookingSummary> {
```

Comment:
Noticed that methods on all our API clients were quite inconsistent as to whether they were marked `async` or not. Added to them all as I assumed they should be 🤔 - does that seem correct?

Reply:
Seems fine to me. I think they're almost not explicitly `async` but actually are through what's calling them so looks fine to me.